src/core/qgselevationmap.h

```diff
@@ -42,31 +42,96 @@ class QgsRasterBlock;
 class CORE_EXPORT QgsElevationMap
 {
   public:
+
+    //! Methods used to select the elevation when two elevation maps are combined
+    enum class CombineMethod
```

Comment:
Could you move this enum to qgis.h instead? (That should be the default place for enums)